be/test/env/env_broker_test.cpp

```diff
@@ -30,17 +30,16 @@ class MockBrokerServer {
     void listPath(TBrokerListResponse& response, const TBrokerListPathRequest& request) {
         std::string path = url_path(request.path);
         TBrokerFileStatus status;
-        bool is_dir = false;
-        Status st = _env->is_directory(path, &is_dir);
-        if (st.is_not_found()) {
+        StatusOr<bool> status_or = _env->is_directory(path);
+        if (status_or.status().is_not_found()) {
             response.opStatus.__set_statusCode(TBrokerOperationStatusCode::FILE_NOT_FOUND);
             return;
-        } else if (!st.ok()) {
+        } else if (!status_or.ok()) {
             response.opStatus.__set_statusCode(TBrokerOperationStatusCode::TARGET_STORAGE_SERVICE_ERROR);
             return;
         }
-        uint64_t size = 0;
-        (void)_env->get_file_size(path, &size);
+        bool is_dir = status_or.value();
+        uint64_t size = _env->get_file_size(path).value();
```

Comment:
```suggestion
        uint64_t size = is_dir ? _env->get_file_size(path).value() : 0;
``` 
This is why the unit tests failed.